scripts/SANS/sans/gui_logic/models/state_gui_model.py

```diff
@@ -367,7 +367,7 @@ def event_binning(self, value):
     # - wavelength_and_pixel_adjustment
     # This is not something that needs to be known at this point, but it is good to know.
     # ------------------------------------------------------------------------------------------------------------------
-    def _update_wavelength(self, min_value=None, max_value=None, step=None, step_type=None):
+    def _update_wavelength(self, min_value=None, max_value=None, step=None, step_type=None, range=None):
```

Comment:
`range` is a reserved keyword in Python, might be a good idea to consider something else here.